.changeset/young-chicken-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/vite-plugin-svelte': major
```

Comment:
Got this warning when I choose "major":
```
🦋  WARNING: Releasing a major version for @sveltejs/vite-plugin-svelte will be its first major release.
🦋  If you are unsure if this is correct, contact the package's maintainers before committing this changeset.
```

Don't think I've seen this before, but maybe it's still safe? Otherwise we can choose minor, and the new release according to semver should still be a breaking change.

Reply:
Seen that before